pkg/codegen/internal/test/testdata/simple-enum-schema/dotnet/Enums.cs

```diff
@@ -0,0 +1,47 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System.Runtime.Serialization;
+
+namespace Pulumi.PlantProvider
+{
+    /// <summary>
+    /// plant container colors
+    /// </summary>
+    public readonly struct ContainerColor : IEquatable<ContainerColor>
+    {
+        private readonly string _value;
+
+        private ContainerColor(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        public static ContainerColor Red { get; } = new ContainerColor("red");
+        public static ContainerColor Blue { get; } = new ContainerColor("blue");
+        public static ContainerColor Yellow { get; } = new ContainerColor("yellow");
+
+        public static bool operator ==(ContainerColor left, ContainerColor right) => left.Equals(right);
+        public static bool operator !=(ContainerColor left, ContainerColor right) => !left.Equals(right);
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override bool Equals(object obj) => obj is ContainerColor other && Equals(other);
+        public bool Equals(ContainerColor other) => string.Equals(_value, other._value, StringComparison.Ordinal);
```

Comment:
`StringComparison` is in the `System` namespace, so you'll need a `using System;` at the top of the file.